CHANGELOG.rst

```diff
@@ -1,6 +1,12 @@
 Release Notes
 *************
 
+1.3.2 (2021-12-13)
+~~~~~~~~~~~~~~~~~~
+* Elaborate the range of years that can be queried `#123 <https://github.com/eWaterCycle/era5cli/pull/123>`_
```

Comment:
```suggestion
* Extend the range of years that can be queried `#123 <https://github.com/eWaterCycle/era5cli/pull/123>`_
```
Would it be nice to have a link here to an ECMWF post explaining the back extension years?

Reply:
I agree, it would be! Unfortunately I couldn't find one. I found an email, but no online version of the same message... Instead of searching further, I'ma push on with the release ;)